string detected_object
int64 x_position
int64 y_position
int64 z_position

